src/Authentication/Actions/EmailActivator.php

```diff
@@ -23,6 +24,13 @@ public function show(): string
             throw new RuntimeException('Cannot get the User.');
         }
 
+        $userEmail = $user->getAuthEmail();
```

Comment:
At this point calling `getAuthEmail()` is the same as just calling `->email`, right? With no trait that needs the `getAuth*` methods do we even need them anymore?

Reply:
Yes, it is the same as calling `getEmail()`, so it is the same as `->email`.

About `getAuth*()` methods, we don't need them now. But when we introduce Interface for *User*, probably we will need them. So I am okay to remove `getAuth*()` now.